### DISCUSSION TITLE
SequelizeDatabaseError

### DISCUSSION THREAD

**Discussion by gba3455** [whyour/qinglong#1760] [Q&A]
大佬，我从v2.12升级到v2.15，旧数据拷贝到新容器的data下，
登录时提示SequelizeDatabaseError SQLITE_READONLY:attempt to write a readonly database
我进bash里chmod 777 database.sqlite后
依然是这个错误

**Comment by gba3455** [comment]
我自己搞定了。。。

> 大佬，我从v2.12升级到v2.15，旧数据拷贝到新容器的data下， 登录时提示SequelizeDatabaseError SQLITE_READONLY:attempt to write a readonly database 我进bash里chmod 777 database.sqlite后 依然是这个错误

**Comment by WinterBoring** [comment]
请问是怎么搞定的，导入数据就是这个错误！谢谢
